### PR TITLE
Switching panorama scripts and playbooks dependencies

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Switched the usage of deprecated Panorama commands in multiple files.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

### CODE REVIEW

**Comment by altmannyarden** [comment]
Yuval, Let's talk about this PR.
Usually we don't change the commands for existing playbooks (BC). We need to be sure it will not happen here

**Comment by YuvHayun** [comment]
> Yuval, Let's talk about this PR. Usually we don't change the commands for existing playbooks (BC). We need to be sure it will not happen here

I talked with @GuyAfik about the deprecated commands, he told me it's the exact same command (up to copy-paste) just with a different prefix

**Comment by GuyAfik** [comment]
a few months ago, we have deprecated all the commands in panorama integration started with `panorama` prefix to use `pan-os` prefix. Those commands are completely equivalent. The only change is the command name, those who start with `panorama` are deprecated and those who start with `pan-os` we want to encourage customers to use them.  

@altmannyarden see this [PR](https://github.com/demisto/content/pull/17632) where the change was done.

File: Packs/ExpanseV2/ReleaseNotes/1_10_14.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (ShahafBenYakir) [inline Packs/ExpanseV2/ReleaseNotes/1_10_14.md:1]:
```suggestion
```

File: Packs/PAN-OS/ReleaseNotes/1_14_1.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (ShahafBenYakir) [inline Packs/PAN-OS/ReleaseNotes/1_14_1.md:1]:
```suggestion
```

File: Packs/PAN-OS/ReleaseNotes/1_14_3.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (ShahafBenYakir) [inline Packs/PAN-OS/ReleaseNotes/1_14_3.md:1]:
```suggestion
```

File: Packs/PANOStoCDLMonitoring/ReleaseNotes/1_0_10.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### PANOStoCortexDataLakeMonitoring
+- Updated the Docker image to: *demisto/python3:3.10.4.30607*.
+- Fixed an issue where the script was using deprecated Panorama commands.
```
Comment (ShahafBenYakir) [inline Packs/PANOStoCDLMonitoring/ReleaseNotes/1_0_10.md:5]:
```suggestion
- Fixed an issue where the script was using deprecated Panorama commands.
- Updated the Docker image to: *demisto/python3:3.10.4.30607*.
```

File: Packs/PaloAltoNetworks_PAN_OS_EDL_Management/ReleaseNotes/1_1_4.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 
 #### Playbooks
-##### PAN-OS - Block IP and URL - External Dynamic List v2
+##### PAN-OS - Block Domain - External Dynamic List
```
Comment (ShahafBenYakir) [inline Packs/PaloAltoNetworks_PAN_OS_EDL_Management/ReleaseNotes/1_1_4.md:3]: ??

**Review by ShahafBenYakir — COMMENTED** [review]
See my comments

File: Packs/CommonPlaybooks/Playbooks/playbook-Block_Account_-_Generic.yml [hunk 6]
```diff
@@ -369,4 +369,4 @@ inputs:
   description: PAN-OS Tag name to apply to the username that you want to block.
 outputs: []
 tests:
-- block_indicators_-_generic_-_test
+- No tests (auto formatted)
```
[the remark this answers is not in the document]
Comment (YuvHayun) [reply Packs/CommonPlaybooks/Playbooks/playbook-Block_Account_-_Generic.yml:372]: this playbook does not appear in the conf.json, I consulted @GuyAfik and he told me I can delete this test.

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**